app/src/main/java/io/pslab/adapters/SensorLoggerListAdapter.java

```diff
@@ -182,6 +194,24 @@ private void populateMapData(SensorDataBlock block) {
                 }
             }
             setMapDataToIntent(array);
+        } else if (block.getSensorType().equalsIgnoreCase(PSLabSensor.ACCELEROMETER)) {
+            RealmResults<AccelerometerData> data = LocalDataLog.with().getBlockOfAccelerometerRecords(block.getBlock());
+            JSONArray array = new JSONArray();
+            for (AccelerometerData d : data) {
+                try {
+                    JSONObject i = new JSONObject();
```

Comment:
Please give relevant variable names

Reply:
I followed the convention that was used in Luxmeter and Barometer.